platform/nativescript/framework.js

```diff
@@ -19,27 +13,6 @@ setVue(Vue)
 Vue.use(ModalPlugin)
 Vue.use(NavigatorPlugin)
 
-const newLineRegExp = /\\n/g
-
-console.log = (function(log, inspect, Vue) {
-  return function(...args) {
-    return log.call(
-      this,
-      ...Array.prototype.map.call(args, function(arg) {
-        return inspect(arg, {
-          depth: 2,
-          colors: Vue.config.debug,
-          showHidden: true
-        }).replace(newLineRegExp, '\n')
-      })
-    )
-  }
-})(console.log, inspect, Vue)
-
-console.keys = function(object) {
-  console.log(Object.keys(object))
```

Comment:
@farfromrefug same with this one - there is no default `console.keys` as far as I'm aware in NativeScript (unless it has been added recently) - so this shouldn't be a problem, but can be made opt-out (under the same umbrella as console.log overrides)

Reply:
@rigor789 again should be opt-in. Especially as you dont have typings for this, right?

Reply:
@farfromrefug that's correct, there are no typings. I was pretty sure this was a standard method on console - but apparently not. In that case I'm happy to remove this one - and move  it to user-land if needed.